.github/labeler.yml

```diff
@@ -0,0 +1,112 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+build:
+  - changed-files:
+    - any-glob-to-any-file:
+      - '.github/**'
+      - 'gradle/**'
+      - '*gradlew*'
+
+clients:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'clients/**'
+
+
+tests:
+- all:
+  - changed-files:
+    - any-glob-to-all-files: 
+      - 'tests/**'
+      - '**/test/**'
+
+
+connect:
+- all:
+  - changed-files:
+    - any-glob-to-any-file: 
+      - 'connect/**/*'
+    - all-globs-to-all-files:
+      - '!connect/mirror/*'
+
+mirror-maker-2:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'connect/mirror/**'
+
+core:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'core/**'
+    - '*server*/**'
+
+storage:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'storage/**'
+
+streams:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'streams/**'
+
+tools:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'tools/**'
+    - 'trogdor/**'
+    - 'vagrant/**'
+    - 'committer-tools/**'
+
+docs:
+- changed-files:
+  - any-glob-to-all-files:
+    - 'docs/**'
+    - '**/*.md'
+    - 'examples/**'
+
+docker:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'docker/**'
+
+
+coordinator:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'group-coordinator/**'
+    - 'coordinator-common/**'
```

Comment:
Could you add `share`-related folder?

Reply:
Hi @chia7712 ,
All comments are addressed in 80bfa4a2f54923f0f07f8195040147e52a3db85a
There is a special case `share-coordinator`, should we put `share` or `coordinator` to it?

Reply:
Ah, we should use KIP-932 for the share coordinator